### DISCUSSION TITLE
Reasons for KFImage requiring iOS 14?

### DISCUSSION THREAD

**Discussion by mpdifran** [onevcat/Kingfisher#1748] [Q&A] [answered]
Hey all!

We still support iOS 13, and have been using KFImage in `KingfisherSwiftUI` v5.15.8. I noticed on `master` KFImage has been set to only support iOS 14 and up.

Is there a reason for this change? If I want to start using `Kingfisher` v6.x.x for the upcoming release of Xcode 13 / iOS 15, is there a way I can continue to support iOS 13?

**Comment by onevcat** [comment, marked as answer]
This is mainly due to issue #1660 and the reason was written there. Kingfisher v6.x's SwiftUI part will continue support from iOS 13, it won't change. The current master contains breaking changes mentioned in the issue above so it will be tag as a major update as version 7.

You can still use v6 for Xcode 13 and iOS 15, it should not affect except you still need that workaround in #1660 sometimes.
